src/engine/main/NetworkManager.C

```diff
@@ -5204,6 +5204,10 @@ NetworkManager::CloneNetwork(const int id)
 //    DirectDatabase route, create a psuedo clone by starting an entirely
 //    new network based off of the cloning target.
 //
+//    Alister Maguire, Mon Mar  9 13:31:50 PDT 2020
+//    Assume we can use the direct route if query atts say yes and there are
+//    no expressions. Also, we can use this route for screen picks.
+//
```

Comment:
if the screen pick require actual data, and the direct route operates on original data, how can screen picks take the direct route?

Reply:
I haven't seen any evidence that "actual data" is needed for a screen pick. The case that I would imagine it being needed is when a transforming operator (that we have the inverse transform for) is applied, but this seems to be handled as is. 
As far as I understand, this is the order of pipeline when using the DirectDatabaseQOT:

- Pick is performed to retrieve element id
- New network is created based on existing one
- DirectDatabaseQOT is added to new network

Since the initial pick is always performed on the actual data, I think the id will always be correct... If you can think of any edge cases I'm missing, please let me know.

Reply:
Thanks for the clarification.